### ISSUE TITLE
Option to include default values in generated output.

### ISSUE THREAD

**Issue by kruncher** [tobiaslabs/json-schema-to-markdown#3]
For instance,
```json
{
  "type": "object",
  "description": "Foo",
  "properties": {
    "meaning": {
      "type": "number",
      "description": "Meaning to the ultimate question.",
      "default": 42
    }
  }
}
```

I think that it would be cool if this produced something like the following:
```markdown
Foo

The schema defines the following properties:

# `meaning` (number)

Meaning to the ultimate question.

Default: 42
```

For object literals and arrays I suppose it could just produce a code block.

**Comment by kruncher** [comment]
See pull request #4

**Comment by saibotsivad** [comment]
Very cool, thanks!

**Comment by saibotsivad** [comment]
Closed with #4 merge
